index.html

```diff
@@ -8,6 +8,7 @@
     />
     <title>Rick and Morty Character Log</title>
     <link rel="stylesheet" href="./styles.css" />
+    <script src="index.js" defer></script>
```

Comment:
If you are using modern JavaScript modules (ES6 modules) that contain **import** and **export** statements, you must specify `type=“module”`. Without this attribute, the import and export statements will not be recognized.

```suggestion
    <script type="module" src="index.js"></script>
```

If you use the `type=“module”` attribute in the  tag, you don’t need to explicitly add the defer attribute. This is because modules are loaded and executed asynchronously by default, similar to using defer.